core-aam/index.html

```diff
@@ -173,22 +173,21 @@ <h2>Introduction</h2>
       </p>
       <ul>
         <li>
-          <cite><a class="specref" href="">Accessible Rich Internet Applications (WAI-ARIA) 1.2</a></cite> [[WAI-ARIA-1.2]], a planned
-          <abbr title="World Wide Web Consortium">W3C</abbr> recommendation, defines the <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> standard.
+          <cite><a class="specref" href="">Accessible Rich Internet Applications (WAI-ARIA) 1.2</a></cite> [[WAI-ARIA-1.2]], a <abbr title="World Wide Web Consortium">W3C</abbr> recommendation,
+          defines the <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> standard.
         </li>
         <li>
           <cite
             ><a href="http://www.w3.org/TR/wai-aria-practices/"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices Guide</a></cite
           >
-          [[WAI-ARIA-PRACTICES-1.2]], a planned W3C Working Group Note, describes how web content developers can develop accessible rich internet applications using
-          <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr>. It provides detailed advice and examples directed primarily to web application developers, yet also useful to user agent
-          and developers of assistive technologies.
+          [[WAI-ARIA-PRACTICES-1.2]], describes how web content developers can develop accessible rich internet applications using <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr>.
+          It provides detailed advice and examples directed primarily to web application developers, yet also useful to user agent and developers of assistive technologies.
         </li>
         <li>
           <cite
             ><a href="http://www.w3.org/TR/wai-aria-roadmap/">Roadmap for Accessible Rich Internet Applications (<abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Roadmap)</a></cite
           >
-          [[WAI-ARIA-ROADMAP]], a planned W3C Working Group Note, defines the path to make rich web content accessible, including steps already taken, remaining future steps, and a time line.
+          [[WAI-ARIA-ROADMAP]], defines the path to make rich web content accessible, including steps already taken, remaining future steps, and a time line.
```

Comment:
@pkra mentioned we should drop this, and I agree, I think we should drop. I'll drop before merging this PR.

Reply:
Not clear about this. Are you both suggesting we should close this PR?